commands.js

```diff
@@ -306,6 +351,7 @@ var commands = exports.commands = {
 	hiddenroom: 'privateroom',
 	secretroom: 'privateroom',
 	privateroom: function (target, room, user, connection, cmd) {
+		if (room.isPersonal) return this.errorReply("Personal rooms configuration can't be changed.");
```

Comment:
Is this necessary? I detailed above why I think public and hidden are valid settings for personal rooms.